server/datastore/mysql/migrations/tables/20220518124708_AddSourceToSoftwareCveTable.go

```diff
@@ -0,0 +1,24 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20220518124708, Down_20220518124708)
+}
+
+func Up_20220518124708(tx *sql.Tx) error {
+	_, err := tx.Exec(
+		"ALTER TABLE `software_cve` ADD COLUMN `source` int DEFAULT '0'",
```

Comment:
Am guessing you could hit the scenario of a CVE affecting e.g. `curl` in macOS/Windows (`source` would be `0`), and `Ubuntu` (`source` would be `1`). And I believe it would fail (or ignored depending on the insert statement) the `UNIQUE KEY `unique_cpe_cve` (`cpe_id`,`cve`),` constraint.

Reply:
That means that we are treating the same version of a software on different platforms as the same thing? If so, that should be fine, in that scenario we don't want to insert duplicates (at the end of the day I think the end-user only cares about the CVE, not were it came from).

Reply:
You have a good point. For MVP this is a-ok. If we _ever_ need to record all the sources we can revisit.